packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1795,24 +1781,32 @@ class OpacityLayer extends ContainerLayer {
       return true;
     }());
 
-    if (enabled)
+    final int realizedAlpha = alpha!;
+    // The type assertions work because the [alpha] setter nulls out the
+    // engineLayer if it would ahve changed type (i.e. changed to or from 255).
```

Comment:
`ahve` -> have

Reply:
Ahh thanks, fixed. I also noticed in the setter above I was accessing the gettr instead of the instance var. Changed that too.